jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -149,6 +149,7 @@
 	/*[PR CMVC 125822] Move RAM class fields onto the heap to fix hotswap crash */
 	private ProtectionDomain protectionDomain;
 	private String classNameString;
+	private String packageName;
```

Comment:
I see classNameString isn't set transient, but many of the other cached variables are. Could mark these as transient to be consistent, although I don't expect it makes any difference since Class is handled specially for serialization.

Reply:
`classNameString` and `packageName` values are consistent after they are computed and could be persisted via serialization. 
It seems there is no specially serialization for `j.l.Class` (maybe I missed something?)

Reply:
These fields are not part of the serialization spec for java.lang.Class and should be transient.

Reply:
Agreed, marked a few other instance fields as `transient`.